software-engineering-policies/NFRs/NFRPolicy.md

```diff
@@ -0,0 +1,21 @@
+# NFRs
+
+Non functional requirements must be seen as everyone's responsibility.
+
+## Scope
+
+This policy seeks to ensure that non functional requirements are present and delivered as part of a piece of work undertaken by a delivery team.
+
+Please also see the UKHO NFR definition [here](https://ukho.sharepoint.com/sites/Technology/eagwiki/Pages/NFR%27s.aspx).
+
+## Implementation
+
+Delivery teams should ensure NFRs are present when starting a new piece of work. If this is not the case, the solution architect aligned with the team should be tasked with discovering and providing the relevant NFRs. If the team does not know who their solution architect is, they should consult with Trevor Rookes.
```

Comment:
suggest Trevor Rookes is replaced with the lead architects position then you won't need to keep it up to date if Trevor leaves...

Reply:
Thanks @gototheant, updated